src/packages/@app/pages/Dashboard/Dashboard.ViewModel.old.ts

```diff
@@ -23,6 +23,10 @@ export class DashboardViewModel {
     await this.teamRepository.setOpenTeam(teamId);
   };
 
+  public getActiveWorkspace = () => {
+    return this.workspaceRepository.getActiveWorkspace();
+  };
```

Comment:
This is old view model, please add this is in new View model.

Reply:
resolved